migrating/checklists/ossm-migrating-premigration-checklists-assembly.adoc

```diff
@@ -0,0 +1,258 @@
+// Module included in the following dir:
+//
+// * service-mesh-docs-main/checklist.adoc
+
+:_mod-docs-content-type: ASSEMBLY
+[id=ossm-migrating-premigration-checklists-assembly]
+= Premigration checklists
+include::_attributes/common-attributes.adoc[]
+:context: ossm-migrating-premigration-checklists-assembly
+
+toc::[]
+
+The checklists make the following assumptions:
```

Comment:
Maybe a bit more strict description here to make it clear users should not start the migration without completing the checklist?

Reply:
Not resolved yet.

Reply:
Ah, a note was added below. Sorry.